packages/front-end/components/Experiment/TabbedPage/SetupTabOverview.tsx

```diff
@@ -43,6 +46,18 @@ export default function SetupTabOverview({
 
   const isBandit = experiment.type === "multi-armed-bandit";
 
+  if (showHypothesisModal) {
+    return (
```

Comment:
question: Should we conditionally render this instead of returning only the modal?

Currently when you open the modal the content on the page shifts as we are not rendering the `Overview` section anymore.

Reply:
Ooof - great catch! This has been updated so there isn't any shift. Thanks!